modules/ROOT/pages/twilio/twilio-connector.adoc

```diff
@@ -1,4 +1,4 @@
-= Twilio Connector
+= Twilio Connector - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
```

Comment:
HIDDEN TEXT
Minor edits; please compare to orig.
Is there no text between the title and this list?

* Get Message List
* Redact Message
* Send Message
These fields can accompany an operation:
[%header,cols="30s,70a"]
|===
|Field |Description
|Account Sid |The unique ID of the account that sent this message
|Date Created |When the resource was created
|Media Sid |Unique ID for the media
|Message Sid |Unique ID for the message, generated after you send a message
|Date Sent |Date the message was sent from Twilio, in RFC 2822 format
|From |Phone number or sender ID that initiated a message, in E.164 format.
|To |Phone number that received a message, in E.164 format
|Entity Reference |MEL expression for what you want to view, such as the payload
|===
For more information about each format, see the See Also section at the end of this document.